client/my-sites/checkout/composite-checkout/components/prepurchase-notices/akismet-product-overlaps-owned-product-notice.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { useSelector } from 'react-redux';
+import { getProductBySlug } from 'calypso/state/products-list/selectors';
+import { getUserPurchases } from 'calypso/state/purchases/selectors';
+import PrePurchaseNotice from './prepurchase-notice';
+
+import './style.scss';
+
+type Props = {
+	overlapingProductSlug: string;
+	cartProductSlug: string;
+};
+
+const AkismetProductOverlapsOwnedProductNotice: FunctionComponent< Props > = ( {
+	overlapingProductSlug,
+	cartProductSlug,
+} ) => {
+	const translate = useTranslate();
+	const products = useSelector( ( state ) => ( {
+		overlaping: getProductBySlug( state, overlapingProductSlug ),
+		cart: getProductBySlug( state, cartProductSlug ),
+	} ) );
+	const purchases = useSelector( getUserPurchases );
+	const purchase = Array.isArray( purchases )
+		? purchases.find( ( p ) => p.productSlug === products.overlaping?.product_slug )
+		: null;
+	const purchaseId = purchase?.id;
+	const subscriptionUrl = purchaseId
+		? `/me/purchases/siteless.akismet.com/${ purchaseId }`
+		: '/me/purchases/';
+
+	const message = translate(
+		'You current own {{oldProduct/}}. The plan you are about to purchase {{newProduct/}} contains all the same functionality and benefits of your current plan. Consider {{link}}removing your {{oldProduct/}} subscription{{/link}} to avoid paying for both plans.',
```

Comment:
As you mentioned elsewhere, this text is a bit lengthy 😅 

WDYT about:
"The plan you are about to purchase contains all the same features of your current plan. Consider removing your (Akismet Old) subscription to avoid paying for both plans."

Reply:
It looks similar to the default Jetpack notice - I don't mind using any of them. But the version you suggests now works for me as well, I think it looks fair and clear.